src/stylesheets/index.css

```diff
@@ -170,10 +170,12 @@ body {
   margin: 0;
 }
 
-h1 {
-  font-size: var(--j2-font-size-heading-1);
-  font-weight: var(--j2-font-weight-strong);
-  line-height: var(--j2-line-height-heading-1);
-  margin: 0;
-  color: var(--j2-color-primary);
+@layer base {
```

Comment:
That's cool - there seems to be a whole new CSS paradigm that I completely missed out on.